src/api-reference/fi-service/financial-integration-service.markdown

```diff
@@ -0,0 +1,1124 @@
+# Financial Integration Service
+
+* [Financial Integration API](#fis)
+  - [GET a List of Financial Transactions based on a search criteria](#gettransactions)
+  - [POST Financial Transactions acknowledgements](#postacknowledgements)
+  - [POST Financial Transactions confirmations](#postconfirmations)
+* [Response Codes](#responsecodes)
+* [Financial Integration sample documents](#examples)
+  - [Expense sample document](#expense)
+  - [Invoice sample document](#invoice)
+  - [Cash Advance sample document](#cashadvance)
+
+## Financial Integration API
+
+The Financial Integration API allows an external system to interact with financial documents generated from Concur, for financial posting into an ERP.
+
+This Web Service provides an automated solution to request available data objects such as approved expense reports, cash advance requests and invoices, to import to the client internal system, with an opportunity to send posting confirmation back into Concur before the object is locked down and cannot be altered in Concur.
+
+Below are some benefits for using the Financial Integration service:
+
+* Using a modern method, no more handling of flat files.
+* Close to real time experience VS SAP Concur over night processing.
+* 1 to 1 Concur to ERP reconciliation VS the extract batch processing.
+* Opportunity to modify and retry before the object is locked in Concur.
+* Insures data sync between systems.
+
+### Version
+
+1.0
+
+### Authorization
+
+The Financial Integration API works only with the new [Authentication API](https://developer.concur.com/api-reference/authentication/apidoc.html).
+
+```
+HEADER:   "Authorization: Bearer {YOUR ACCESS TOKEN}"
+```
+
+## GET a list of Financial Transactions based on a search criteria
+
+Making a GET request to the financial transactions API will return a list of available financial documents, ready to be processed.
+
+```
+GET   /financialintegration/fi/v1/companies/{companyId}/transactiontypes/{docType}/transactions
+```
+
+### Parameters
+
+| Name | Type | Format | Description |
+| --- | --- | --- | --- |
+| `companyId` | `string` | `path` | **Required** Company Outtask Id in Concur |
+| `docType` | `string` | `path` | **Required** The financial document type to return. VALUES: expense; invoice; cashadvance. Only one type of transactions can be retrieved at a time |
+| `OuttaskCompanyId` | `string` | `header` | **Required** Outtask Company Id in Concur |
```

Comment:
Technically speaking, should not be making the `OuttaskCompanyId` available either.